config/routes.rb

```diff
@@ -13,6 +13,8 @@
 
     get "/subscribers/:address/subscriptions", to: "subscribers#subscriptions",
       constraints: { address: /.+@.+\..+/ }
+    post "/subscribers/:address/change-address", to: "subscribers#change_address",
+      constraints: { address: /.+@.+\..+/ }
```

Comment:
I think it's possible to move this constraint into a block and then every route underneath it will have the same constraint applied to it.

Reply:
Yup works a treat